### PR TITLE
Fix Makefile purge

### PR DESCRIPTION
**Pull request by Coder-256**
I had an old `/usr/local/lib/libr2.1.4.0.dylib` file on my computer.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -321,6 +321,7 @@ purge: purge-doc purge-dev user-uninstall
 	rm -f "${DESTDIR}${BINDIR}/r2"
 	rm -f "${DESTDIR}${LIBDIR}/libr_"*
 	rm -f "${DESTDIR}${LIBDIR}/libr2.${EXT_SO}"
+	rm -f "${DESTDIR}${LIBDIR}/libr2."*".${EXT_SO}"
```
Comment (radare) [inline Makefile]: just `rm -f "${DESTDIR}${LIBDIR}/libr2.*${EXT_SO}"` should be enough

Comment (Coder-256) [reply Makefile]: Well if you want to compress it into one line, I think that `rm -f "${DESTDIR}${LIBDIR}/libr2."*".${EXT_SO}"` would actually be a better alternative. Plus you cannot put a glob (such as the star) inside quotes.

Comment (radare) [reply Makefile]: Do the change and ill merge

Comment (Coder-256) [reply Makefile]: Done.

File: Makefile [hunk 2]
```diff
@@ -321,6 +321,7 @@ purge: purge-doc purge-dev user-uninstall
 	rm -f "${DESTDIR}${BINDIR}/r2"
 	rm -f "${DESTDIR}${LIBDIR}/libr_"*
 	rm -f "${DESTDIR}${LIBDIR}/libr2.${EXT_SO}"
```
Comment (radare) [inline Makefile]: you can now remove this line

**Comment by radare** [comment]
hello?

**Comment by Coder-256** [comment]
Ok done.

**Pull request merged**